test/container/Dockerfile-Ubuntu

```diff
@@ -6,6 +6,7 @@ RUN apt-get update -y -qq && apt-get upgrade -y -qq && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends -o Dpkg::Use-Pty=0 \
     asciidoc \
     astyle \
+    bcachefs-tools \
```

Comment:
bcachefs-tools was removed from Debian unstable and the future is unclear.

Reply:
@bdrung what do you suggest we do ? 

As you can see I did not add it to the Debian container and I certainly hope we can have at least one container with bcachefs tools (it does not have to be Ubuntu).

If the CI container no longer builds due some distro packaging changes, it is trivial to remove.

There seems to be interest for a dedicated bcachefs dracut module also - https://github.com/dracut-ng/dracut-ng/issues/145

Reply:
I agree here, as it is, bcachefs is experimental and churning too much for use to test. I would leave it out for now. 
I think we're going to want to test ext4; xfs; and btrfs, as those are the three commonly used filesystems.

Reply:
I am fine with adding bcachefs-tools to the Ubuntu container, since it can be easily dropped when needed.

Reply:
For now I removed bcachefs in the interest of making progress first with other fs types, like xfs .